api/database/helpers/ApiEnums.php

```diff
@@ -20,11 +20,12 @@ public static function operationalRequirements() : array
             'DRIVERS_LICENSE',
             'WORK_WEEKENDS',
             'OVERTIME_SCHEDULED',
-            'OVERTIME_SHORT_NOTICE'
+            'OVERTIME_SHORT_NOTICE',
+            'OVERTIME_OCCASIONAL',
+            'OVERTIME_REGULAR',
```

Comment:
Larger proposal aside, it feels worth asking whether it's alright if all the types of overtime show up in forms. If so, then we could just merge this as-is, and leave the more clever solution for later :)

Reply:
added more subselection stuff
given the enum group may remain in flux for a while or tied to a feature_flag, this might be a compromise for now

Reply:
test by navigating the app I suppose? specifically where Operational Requirements are in play